packages/mobile/src/alert/reducer.ts

```diff
@@ -2,9 +2,14 @@ import { Actions, ActionTypes } from 'src/alert/actions'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import { RootState } from 'src/redux/reducers'
 
+export enum ErrorDisplayType {
+  'BANNER' = 0,
```

Comment:
```suggestion
export enum ErrorDisplayType {
BANNER,
INLINE
}
```

enum will default to enumerating 0 1 2 etc

Reply:
good to know! thanks